src/hasher.cpp

```diff
@@ -121,28 +86,14 @@ namespace libtorrent
 #elif TORRENT_USE_CRYPTOAPI
 	hasher::hasher(hasher const& h)
```

Comment:
in fact, you probably don't need to define the copy constructor explicitly here. but if you do, you can just define it as ``= default;``